backend/src/services/wallet_service.py

```
@@ -59,43 +78,85 @@ async def get_wallet_info(self) -> Optional[Dict]:
+                        price_data = price_response.json().get('data', {})
+
+                        sol_price_info = price_data.get('So11111111111111111111111111111111111111112')
+                        if sol_price_info:
+                            sol_price = float(sol_price_info['price'])
+
+                        for token in tokens:
+                            token_price_info = price_data.get(token['mint_address'])
+                            if token_price_info:
+                                token['price_usd'] = float(token_price_info['price'])
+                                token['usd_value'] = token['balance'] * token['price_usd']
+            except Exception as pe:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)